Create postbuild

ArthurMuehl opened the pull request:
I am adding this postbuild file on behalf of Christina Viehmann. Please merge the repositories after checking its content. Thank you!

ArthurMuehl committed:
Create postbuild

I am adding this postbuild file on behalf of Christina Viehmann. Please merge the repositories after checking its content. Thank you!

yfiua commented:
Hi @rgaiacs , is it supposed to be postBuild with capitalized B? or it does not matter?

rgaiacs commented:
It **MUST** be `postBuild` with capitalized `B`.

@ArthurMuehl please change the name of the file and run

```bash
git add postBuild
git rm postbuild
git commit --amend
```

ArthurMuehl committed:
Rename postbuild to postBuild

I have renamed the file to ensure functionality. Thank you for catching my error!

ArthurMuehl commented:
I have renamed the file to ensure functionality. Thank you for catching my error!
